qt/widgets/common/src/FunctionBrowser.cpp

```diff
@@ -2171,7 +2250,8 @@ Mantid::API::IFunction_sptr FunctionBrowser::getGlobalFunction() {
     multiFun->setDomainIndex(i, i);
     auto fun1 = multiFun->getFunction(i);
     for (size_t j = 0; j < fun1->nParams(); ++j) {
-      QString parName = QString::fromStdString(fun1->parameterName(j));
+      auto parameterName = fun1->parameterName(j);
+      QString parName = QString::fromStdString(parameterName);
```

Comment:
Could both strings be `const`? It looks a bit strange that the same string is duplicated as C-string and `QString` here, but I guess it cannot be helped.